### ISSUE TITLE
Add G-mean as an evaluation metric

### ISSUE THREAD

**Issue by AndriyMulyar** [openml/OpenML#872]
Add the geometric mean (g_mean) of precision and recall as an evaluation metric for classification.
[Geometric Mean](https://imbalanced-learn.readthedocs.io/en/stable/generated/imblearn.metrics.geometric_mean_score.html)

**Comment by joaquinvanschoren** [comment]
Is that different from the harmonic mean? We call that the f_measure:
https://www.openml.org/a/evaluation-measures/f-measure

Hmm, I guess it is a bit different. You can add new evaluation metrics to OpenML by adding a new evaluation engine, yet this is not very well documented at the moment.

**Comment by AndriyMulyar** [comment]
Yes. Here is a nice paper with discussion: [Kubat,1997](https://sci2s.ugr.es/keel/pdf/algorithm/congreso/kubat97addressing.pdf) for anyone who may come across this in the future.
